tests/assembly/f16-f128-windows-x86_64-abi.rs

```diff
@@ -0,0 +1,23 @@
+//@ assembly-output: emit-asm
```

Comment:
Could you make the file name start with `x86_64-windows`? Seems like that would be consistent with some other tests in https://github.com/rust-lang/rust/tree/bf8798c1ba16e75b8c205bd4e1d93d7ff09d13d6/tests/assembly

Reply:
I guess this could also just become an ABI test for all floats if you are up for adding f32 and f64.